src/components/Query.ts

```diff
@@ -222,8 +230,16 @@ export class APIRequest {
         cancelToken: source.token
       })).data
     } catch (err) {
-      const message = err?.response?.data?.message || err.toString()
-      throw new Error(message)
```

Comment:
this could just be `throw new Error(message.toString())` and have the same effect without the type checking

Reply:
same above